gulpfile.js

```diff
@@ -151,8 +151,9 @@ gulp.task('html', ['config', 'styles', 'scripts', 'flash'], function (done) {
             .pipe(jsFilter)
             .pipe($.sourcemaps.init())
             .pipe($.replace('_GIT_REVISION_GOES_HERE_',revision))
-            .pipe($.ngAnnotate())
```

Comment:
`ngAnnotate` was causing problems during code uglifying. Added `mangle: false` to `uglify` to prevent breaking Angular DI.